tests/fv3_conf/fv3_qsub.IN_wcoss2

```diff
@@ -15,7 +15,7 @@ echo -n " $( date +%s )," >  job_timestamp.txt
 cd $PBS_O_WORKDIR
 
 set +x
-module use $( pwd -P )
+module use modulefiles
```

Comment:
These should be absolute paths so that they'll still work if the directory changes.
```suggestion
module use $PWD/modulefiles
```